### PR TITLE
API 5.7

### PR DESCRIPTION
**Pull request by harshil21**
### Checklist for PRs

- [x] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [x] Created new or adapted existing unit tests
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs

### If the PR contains API changes (otherwise, you can delete this passage)

* New classes:
    - [ ] Added `self._id_attrs` and corresponding documentation
    - [ ] `__init__` accepts `**_kwargs`
    
* Added new shortcuts:
    - [ ] In `Chat` & `User` for all methods that accept `chat/user_id`
    - [ ] In `Message` for all methods that accept `chat_id` and `message_id`
    - [ ] For new `Message` shortcuts: Added `quote` argument if methods accepts `reply_to_message_id`
    - [ ] In `CallbackQuery` for all methods that accept either `chat_id` and `message_id` or `inline_message_id`
    
* If relevant:

    - [ ] Added new constants at `telegram.constants` and shortcuts to them as class variables
    - [ ] Added new handlers for new update types
    - [ ] Added new filters for new message (sub)types
    - [x] Added or updated documentation for the changed class(es) and/or method(s)
    - [x] Updated the Bot API version number in all places: `README.rst` and `README_RAW.rst` (including the badge), as well as `telegram.constants.BOT_API_VERSION`
    - [ ] Added logic for arbitrary callback data in `tg.ext.Bot` for new methods that either accept a `reply_markup` in some form or have a return type that is/contains `telegram.Message`

Used the `potato doge` webm sticker from: https://t.me/TrendingStickers/1978

Note about a sticker test:
Didn't implement a `test_bot_methods_3_webm` since the webm file is too large to be a thumb... can anyone compress it? 

Edit:
Skipped that test, see that test for the reason.

### CODE REVIEW

**Review by Poolitzer — APPROVED** [review]
Changes look good!

File: telegram/bot.py [hunk 1]
```diff
@@ -4836,13 +4837,19 @@ def create_new_sticker_set(
...
-                optional): TGS animation with the sticker,
-                uploaded using multipart/form-data. See
-                https://core.telegram.org/animated_stickers#technical-requirements for technical
+                optional): **TGS** animation with the sticker, uploaded using multipart/form-data.
+                See https://core.telegram.org/stickers#animated-sticker-requirements for technical
                 requirements.
 
                 .. versionchanged:: 13.2
                    Accept :obj:`bytes` as input.
+            webm_sticker (:obj:`str` | `filelike object` | :obj:`bytes` | :class:`pathlib.Path`, \
```
Comment (Bibo-Joshi) [inline telegram/bot.py:4846]:
```suggestion
            webm_sticker (:obj:`str` | :term:`file object` | :obj:`bytes` | :class:`pathlib.Path`, \
```
same everywhere wher you *added* this  - just makes it easier to integrate the changes into v14

File: tests/test_sticker.py [hunk 2]
```diff
@@ -30,9 +30,8 @@
 
 @pytest.fixture(scope='function')
 def sticker_file():
-    f = open('tests/data/telegram.webp', 'rb')
-    yield f
-    f.close()
+    with open('tests/data/telegram.webp', 'rb') as f:
+        yield f
```
Comment (Bibo-Joshi) [inline tests/test_sticker.py:34]: if this is not really necessary for this PR, please revert - spares merging trouble :)

File: tests/test_sticker.py [hunk 3]
```diff
@@ -43,9 +42,8 @@ def sticker(bot, chat_id):
 
 @pytest.fixture(scope='function')
 def animated_sticker_file():
-    f = open('tests/data/telegram_animated_sticker.tgs', 'rb')
-    yield f
-    f.close()
+    with open('tests/data/telegram_animated_sticker.tgs', 'rb') as f:
```
Comment (Bibo-Joshi) [inline tests/test_sticker.py:45]: .

File: tests/test_sticker.py [hunk 4]
```diff
@@ -338,18 +362,32 @@ def animated_sticker_set(bot):
...
+    return ss
+
+
 @pytest.fixture(scope='function')
 def sticker_set_thumb_file():
-    f = open('tests/data/sticker_set_thumb.png', 'rb')
-    yield f
-    f.close()
+    with open('tests/data/sticker_set_thumb.png', 'rb') as f:
+        yield f
```
Comment (Bibo-Joshi) [inline tests/test_sticker.py:382]: .

File: tests/test_sticker.py [hunk 5]
```diff
@@ -367,14 +406,38 @@ def test_de_json(self, bot, sticker):
...
+        test_by = f"test_by_{bot.username}"
+        for sticker_set in [test_by, f'animated_{test_by}', f'video_{test_by}']:
+            try:
+                bot.get_sticker_set(sticker_set)
+            except BadRequest as e:
+                if not e.message == "Stickerset_invalid":
+                    raise e
+
+                # We already have static and animated sticker sets, this is just for creating
+                # a video sticker set
```
Comment (Bibo-Joshi) [inline tests/test_sticker.py:425]: If this is only for the video stickers, why loop over all 3 cases above? I'd suggest to either keep it for just video or extend to all three - don't have a preference

File: tests/test_sticker.py [hunk 6]
```diff
@@ -367,14 +406,38 @@ def test_de_json(self, bot, sticker):
         assert sticker_set.name == name
         assert sticker_set.title == self.title
         assert sticker_set.is_animated == self.is_animated
+        assert sticker_set.is_video == self.is_video
         assert sticker_set.contains_masks == self.contains_masks
         assert sticker_set.stickers == self.stickers
         assert sticker_set.thumb == sticker.thumb
 
+    def test_create_sticker_set(self, bot, chat_id, video_sticker_file):
+        """Creates the sticker set (if needed) which is required for tests"""
```
Comment (Bibo-Joshi) [inline tests/test_sticker.py:415]:
```suggestion
        """Creates the sticker set (if needed) which is required for tests.
        Make sure that this test comes before the tests that actually use the sticker sets!
        """
```

**Review by Bibo-Joshi — CHANGES_REQUESTED** [review]
Thanks for the fast PR! Left a few nitpicks.
One can download the thumb of a sticker set - maybe that suffices for the `_3` test?

```python
import asyncio

from telegram import Bot


async def main():
    async with Bot(TOKEN') as bot:
        sticker_set = await bot.get_sticker_set('Potato')
        await (await sticker_set.thumb.get_file()).download('video_thumb.webm')

if __name__ == '__main__':
    asyncio.run(main())
```

File: telegram/bot.py (hunk 1 above)
Comment (harshil21) [reply telegram/bot.py:4846]: this generates warnings when building. (term not in glossary). Ignore them?

Comment (Bibo-Joshi) [reply telegram/bot.py:4846]: ah, because we're not using intersphinx on master yet. yea, I'm okay with ignoring

**Review by Bibo-Joshi — APPROVED** [review]

**Pull request merged**